daemon/loop.go

```diff
@@ -149,6 +150,52 @@ func (d *Daemon) pullAndSync(logger log.Logger) {
 		}); err != nil {
 			logger.Log("err", err)
 		}
+
+		// Find notes in revisions.
+		for i, rev := range revisions {
+			// Skip last revision (since we requested HEAD~1 above)
+			if i == len(revisions)-1 {
+				continue
+			}
+			n, err := working.GetNote(rev)
+			if err != nil {
+				logger.Log("err", errors.Wrap(err, "loading notes from repo; possibly no notes"))
+				continue
+			}
+
+			// If one of the notes has a release event, send that to the service
+			if n.Spec.Type == update.Images {
+				// Map new note.Spec into ReleaseSpec
+				spec := n.Spec.Spec.(update.ReleaseSpec)
+				status := update.ReleaseStatusSuccess
+				if n.Result.Error() != "" {
+					status = update.ReleaseStatusFailed
+				}
+				// And create a release event
+				releaseEvent := update.Release{
+					StartedAt: started,
```

Comment:
I think this is a good candidate for field tweaking then :) Can you either make sure it's in the right place, or file an issue so we don't assume it's all fine because it has a value ..